lib/python/rose/gtk/run.py

```diff
@@ -30,6 +31,17 @@
 def run_suite(*args):
     """Run "rose suite-run [args]" with a GTK dialog."""
 
+    # Check that we are not using Cylc 8: Else return a helpful message:
+    cylc_version = check_output(['cylc', 'version'])
```

Comment:
There is an existing interface for fetching the Cylc version.

lib/python/rose/suite_engine_procs/cylc.py::get_version